Library/Formula/openvpn.rb

```diff
@@ -25,12 +35,22 @@ def install
         "security/pam_appl.h", "pam/pam_appl.h"
     end
 
+    opts = []
+    if build.with? "pkcs11"
+        opts << "--enable-pkcs11"
+        ENV.append_path "PKG_CONFIG_PATH", "/usr/local/lib/pkgconfig"
+        ENV.append "PKCS11_HELPER_CFLAGS", `pkg-config --cflags libpkcs11-helper-1`.chomp
```

Comment:
Same notation here about tabs rather than spaces, and we'd prefer to use:

```ruby
ENV.append "PKCS11_HELPER_CFLAGS", "-I#{Formula["pkcs11-helper"].opt_include}"
```
here.